doc/modules/model_evaluation.rst

```diff
@@ -1756,6 +1759,134 @@ the same does a lower Brier score loss always mean better calibration"
     and probability estimation." <https://drops.dagstuhl.de/opus/volltexte/2008/1382/>`_
     Dagstuhl Seminar Proceedings. Schloss Dagstuhl-Leibniz-Zentrum fr Informatik (2008).
 
+.. _class_likelihood_ratios:
+
+Class likelihood ratios
+-----------------------
+
+The :func:`class_likelihood_ratios` function computes the `positive and negative
+likelihood ratios
+<https://en.wikipedia.org/wiki/Likelihood_ratios_in_diagnostic_testing>`_ for
+binary classes, which are further metrics derived from the
+:func:`confusion_matrix` and that can be interpreted in terms of the pre-test
+and post-test probabilities:
+
+.. math::
+
+   \text{sensitivity} = \frac{tp}{tp + fn},
+
+.. math::
+
+   \text{specificity} = \frac{tn}{tn + fp},
+
+.. math::
+
+   LR_+ = \frac{\text{sensitivity}}{1 - \text{specificity}} = \frac{tp \times
+   (tn + fp)}{fp \times (tp + fn)},
+
+.. math::
+
+   LR_- = \frac{1 - \text{sensitivity}}{\text{specificity}} = \frac{fn \times
+   (tn + fp)}{tn \times (tp + fn)}.
```

Comment:
I think that we should give the mathematical expression only below, and first focus on what the different values mean and how to interpret them. 

In scikit-learn docs, we tend to downplay mathematical formula, and push up meaning and intuitions.

Reply:
We should rather start by way we would want to use this metric. And the reason is: because it is invariant to class prevalence and can be extrapolated between population with different class prevalence. I'll share with you privately a text that I wrote on this.